### PR TITLE
feat: create alert component

### LINKED ISSUE #41: Alert: Create alert component
[Figma design](https://www.figma.com/file/msxmKajRPLYiDDmsP60Ndh/Design-System-v1.1?type=design&node-id=13-36911&mode=design&t=njl1Nz2KssQHhu64-4)

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #41

### CODE REVIEW

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.1'
+lockfileVersion: '6.0'
```
Comment (tewshi) [inline pnpm-lock.yaml:1]: are we downgrading the lock file version?
or is 6.1 not latest?

**Review by tewshi — APPROVED** [review]
lgtm!

File: example/src/views/HomeView.vue [hunk 2]
```diff
@@ -864,6 +945,29 @@ const textFields = ref([
...
+        :on-action="
+          alert.clicks !== undefined &&
+          (() => {
+            alert.clicks++;
+          })
+        "
+        :title="`${alert.type} ${
+          alert.clicks !== undefined ? `(${alert.clicks})` : ''
+        } ${alert.closed ? '(Closed)' : ''}`"
+        :on-close="alert.closeable && (() => (alert.closed = true))"
```
Comment (tewshi) [inline example/src/views/HomeView.vue:966]: can we replace `on-action` and `on-close` with events emitted from the alert component and then use `@action=` and `@close=` event handlers

Comment (tewshi) [reply example/src/views/HomeView.vue:966]: other than this, it's all good.

File: package.json [hunk 3]
```diff
@@ -75,6 +75,7 @@
     "jsdom": "22.1.0",
     "lint-staged": "13.2.2",
     "postcss": "8.4.23",
+    "postcss-color-mod-function": "3.0.3",
```
Comment (kelsos) [inline package.json:78]: image

can we find an alternative here?

Comment (lukicenturi) [reply package.json:78]: removed this, I just figured out we can do it in css directly
https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/color-mix.
What an era to be alive 🤣

File: src/components/alerts/Alert.stories.ts [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+    variant: {
+      control: 'select',
+      options: ['default', 'filled', 'outlined'],
+      table: { category: 'State' },
+    },
+    actionText: { control: 'text' },
+  },
+};
+
+export const Default = {
```
[the remark this answers is not in the document]
Comment (lukicenturi) [reply src/components/alerts/Alert.stories.ts:34]: I'm playing with the typescript now, but haven't succeed yet.
can we do this in next PR?

**Review by kelsos — APPROVED** [review]
lgtm

**Pull request merged**